### PR TITLE
Leap 16.0 wallpaper refresh

### PR DESCRIPTION
**Pull request by lkocman**
Unified 4096x4096 Leap 16.0 wallpaper
    
* Graphics from @hellcp. Many thanks for all the work!
* Resolves Leap part of gh#openSUSE/branding#135
* Covers both dark and light variant
* Use single 4096x4096 file as mentioned in gh#openSUSE/branding#161
* Use the dark 800x600 also for the back.png
* Use default.png instead of default-light.png to maximize compatability.

screenshot

### CODE REVIEW

**Comment by hellcp** [comment]
My only feedback is that we could use this as an opportunity to introduce light/dark auto switching wallpaper in GNOME, however it can be done later down the line

**Comment by lkocman** [comment]
@hellcp shall I simply add the second variant too? Add -light -dark suffixes?

**Comment by hellcp** [comment]
I think GNOME has an xml format that defines light and dark wallpapers that we would need to package

**Comment by lkocman** [comment]
Righto alternatively 

gsettings set org.gnome.desktop.background picture-uri "file:///path/to/image
gsettings set org.gnome.desktop.background picture-uri-dark "file:///path/to/image

here's the xml template https://github.com/GNOME/gsettings-desktop-schemas/blob/master/schemas/org.gnome.desktop.background.gschema.xml.in

I'm just worried about setting light/dark as default, because of openQA

**Comment by hellcp** [comment]
openQA would have to change the theme, does it in any tests? that would already affect the needles in a significant way

**Comment by lkocman** [comment]
Well I thought that we'd go with this as a default bg, then we'd have to have set of two needles.

**Comment by hellcp** [comment]
There is going to be only one wallpaper visible to openQA if only one theme is selected during the install. The default theme is light

**Comment by lkocman** [comment]
Uff I noticed that I probably used mm rather than px. Will fix :-) We should not have 14k resolution wallpapers :-))) (It crashes eog btw)

File: raw-theme-drop/desktop-dark-1600x1200.svg
Comment (hellcp) [inline raw-theme-drop/desktop-dark-1600x1200.svg:1]: Looks different from the rest

File: raw-theme-drop/back.svg
Comment (hellcp) [inline raw-theme-drop/back.svg:1]: We can use the same backdrop here too tbh

Comment (lkocman) [reply raw-theme-drop/back.svg:1]: Will do. Thank you!

**Comment by lkocman** [comment]
@hellcp I like the simplicity of the single color in day variant (normal blend mode)

But I do have a little dilemma.
Here are two colors (I took the darker yellow from the dark variant). The fbbb4bff one is imho more warmer and pleasant to look in later hours. The lighter one is too shiny to watch at in darker room.

PNGs: for convenience
f3ca5eff

fbbb4bff

SVG: 
desktop-3840x2400

What do you think? (Also I'll move it bit down, so there is no black border at the bottom).

**Comment by hellcp** [comment]
I'm a bit concerned with how dark the column outlines look themselves

**Comment by lkocman** [comment]
> I'm a bit concerned with how dark the column outlines look themselves

So do ou prefer the original blend (I believe it was set to light)?

File: raw-theme-drop/desktop-dark-1600x1200.svg
Comment (lkocman) [reply raw-theme-drop/desktop-dark-1600x1200.svg:1]: @hellcp Would you prefer zoom out than crop for different aspect ratio? That's fine too. I just wanted to make sure there won't be huge amount of free space on the top. Same applies for 5:4 and 4:3. With previous we kinda did the zoom out trick with more blank (read yellow or brown) space on top

Comment (lkocman) [reply raw-theme-drop/desktop-dark-1600x1200.svg:1]: Hopefully fixed in last commit.

File: raw-theme-drop/back.svg
Comment (lkocman) [reply raw-theme-drop/back.svg:1]: Done, I used dark. I wonder if we should put "Leap" somewhere on the back.svg

**Comment by lkocman** [comment]
Updated the initial post with screenshot of submitted wallpapers

**Comment by hellcp** [comment]
Did you have an opportunity to check if there's any significant difference between how rsvg and inkscape render those images? Otherwise, if you could please save the svgs as optimized svg in inkscape to make them a bit smaller

**Comment by lkocman** [comment]
Will do @hellcp !

**Comment by lkocman** [comment]
Final distro-specific abstract wallpaper was updated in the summary

File: Makefile [hunk 1]
```diff
@@ -108,7 +113,7 @@ CLEAN_DEPS+=yast.d_clean
 
 plymouth.d:
 	mkdir -p openSUSE/plymouth
-	cp -r plymouth/config/plymouthd.defaults openSUSE/plymouth
```
Comment (lkocman) [inline Makefile:111]: Adopted from Tumbleweed

File: Makefile [hunk 2]
```diff
@@ -133,10 +138,8 @@ install:
 	mkdir -p $(DESTDIR)/usr/share/grub2/themes/${THEME} ${DESTDIR}/boot/grub2/themes/${THEME}
 	cp -a openSUSE/grub2/theme/* ${DESTDIR}/usr/share/grub2/themes/${THEME}
 	perl -pi -e "s/THEME_NAME/${THEME}/" ${DESTDIR}/usr/share/grub2/themes/${THEME}/activate-theme
-	# Plymouth theme
```
Comment (lkocman) [inline Makefile:136]: Adopted from Tumbleweed

File: gnome/openSUSE-default-static.xml [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
+<?xml version="1.0"?>
+<!DOCTYPE wallpapers SYSTEM "gnome-wp-list.dtd">
+<wallpapers>
+  <wallpaper deleted="false">
+    <name>Default Background</name>
+    <filename>/usr/share/wallpapers/openSUSEdefault/contents/images/default.png</filename>
+    <filename-dark>/usr/share/wallpapers/openSUSEdefault/contents/images/default-dark.png</filename-dark>
+    <options>zoom</options>
+    <shade_type>solid</shade_type>
+    <pcolor>#fbc75dff</pcolor>
```
Comment (lkocman) [inline gnome/openSUSE-default-static.xml:10]: Used Leap light bg color as primary, dark bg as secondary

**Comment by lkocman** [comment]
Makefile builds

```
📦[lkocman@osc branding_leap16]$ make
echo "Make sure to have rsvg-view, GraphicsMagick and optipng installed"
Make sure to have rsvg-view, GraphicsMagick and optipng installed
mkdir -p ~/.fonts openSUSE/gfxboot/data-boot/ openSUSE/gfxboot/data-install
cp gfxboot/SourceSansPro-Light.ttf ~/.fonts
for name in back welcome on off glow; do \
	rsvg-convert raw-theme-drop/${name}.svg -w 800 -a -o tmp-gfxboot.d.png; \
	gm convert -quality 100 -interlace None -colorspace YCbCr -geometry 800x600 -sampling-factor 2x2 tmp-gfxboot.d.png openSUSE/gfxboot/data-install/${name}.jpg; \
	rm tmp-gfxboot.d.png; \
done
rsvg-convert gfxboot/text.svg -w 114 -a -o tmp-gfxboot.d.png
gm convert -quality 100 -interlace None -colorspace YCbCr -sampling-factor 2x2 tmp-gfxboot.d.png openSUSE/gfxboot/data-install/text.jpg
rm tmp-gfxboot.d.png
rsvg-convert raw-theme-drop/back.svg -w 800 -a -o tmp-gfxboot.d.png
gm convert -quality 100 -interlace None -colorspace YCbCr -geometry 800x600 -sampling-factor 2x2 tmp-gfxboot.d.png openSUSE/gfxboot/data-boot/back.jpg
rm tmp-gfxboot.d.png
rm ~/.fonts/SourceSansPro-Light.ttf
mkdir -p openSUSE/gnome
sed "s:@VERSION@:16.0:g;s:@GNOME_STATIC_DYNAMIC@:static:g" gnome/wallpaper-branding-openSUSE.xml.in > openSUSE/gnome/wallpaper-branding-openSUSE.xml
cp gnome/openSUSE-default-static.xml openSUSE/gnome/openSUSE-default-static.xml
mkdir -p openSUSE/grub2
cp -a boot/grub2/theme openSUSE/grub2/
rm -rf openSUSE/icewm
mkdir -p openSUSE/icewm
cp -av icewm openSUSE/
'icewm/theme' -> 'openSUSE/icewm/theme'
'icewm/themes' -> 'openSUSE/icewm/themes'
'icewm/themes/yast-installation' -> 'openSUSE/icewm/themes/yast-installation'
'icewm/themes/yast-installation/LICENSE' -> 'openSUSE/icewm/themes/yast-installation/LICENSE'
'icewm/themes/yast-installation/closeA.xpm' -> 'openSUSE/icewm/themes/yast-installation/closeA.xpm'
'icewm/themes/yast-installation/closeO.xpm' -> 'openSUSE/icewm/themes/yast-installation/closeO.xpm'
'icewm/themes/yast-installation/default.theme' -> 'openSUSE/icewm/themes/yast-installation/default.theme'
'icewm/themes/yast-installation/frameAB.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameAB.xpm'
'icewm/themes/yast-installation/frameABL.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameABL.xpm'
'icewm/themes/yast-installation/frameABR.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameABR.xpm'
'icewm/themes/yast-installation/frameAL.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameAL.xpm'
'icewm/themes/yast-installation/frameAR.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameAR.xpm'
'icewm/themes/yast-installation/frameAT.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameAT.xpm'
'icewm/themes/yast-installation/frameATL.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameATL.xpm'
'icewm/themes/yast-installation/frameATR.xpm' -> 'openSUSE/icewm/themes/yast-installation/frameATR.xpm'
'icewm/themes/yast-installation/icons' -> 'openSUSE/icewm/themes/yast-installation/icons'
'icewm/themes/yast-installation/icons/app_11x11.xpm' -> 'openSUSE/icewm/themes/yast-installation/icons/app_11x11.xpm'
'icewm/themes/yast-installation/icons/folder_11x11.xpm' -> 'openSUSE/icewm/themes/yast-installation/icons/folder_11x11.xpm'
'icewm/themes/yast-installation/icons/xterm.xpm' -> 'openSUSE/icewm/themes/yast-installation/icons/xterm.xpm'
'icewm/themes/yast-installation/maximizeA.xpm' -> 'openSUSE/icewm/themes/yast-installation/maximizeA.xpm'
'icewm/themes/yast-installation/maximizeO.xpm' -> 'openSUSE/icewm/themes/yast-installation/maximizeO.xpm'
'icewm/themes/yast-installation/menuButtonA.xpm' -> 'openSUSE/icewm/themes/yast-installation/menuButtonA.xpm'
'icewm/themes/yast-installation/menubg.xpm' -> 'openSUSE/icewm/themes/yast-installation/menubg.xpm'
'icewm/themes/yast-installation/menusel.xpm' -> 'openSUSE/icewm/themes/yast-installation/menusel.xpm'
'icewm/themes/yast-installation/minimizeA.xpm' -> 'openSUSE/icewm/themes/yast-installation/minimizeA.xpm'
'icewm/themes/yast-installation/minimizeO.xpm' -> 'openSUSE/icewm/themes/yast-installation/minimizeO.xpm'
'icewm/themes/yast-installation/taskbar' -> 'openSUSE/icewm/themes/yast-installation/taskbar'
'icewm/themes/yast-installation/taskbar/collapse.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/collapse.xpm'
'icewm/themes/yast-installation/taskbar/desktop.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/desktop.xpm'
'icewm/themes/yast-installation/taskbar/expand.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/expand.xpm'
'icewm/themes/yast-installation/taskbar/linux_backup.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/linux_backup.xpm'
'icewm/themes/yast-installation/taskbar/start.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/start.xpm'
'icewm/themes/yast-installation/taskbar/taskbarbg.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/taskbarbg.xpm'
'icewm/themes/yast-installation/taskbar/taskbuttonactive.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/taskbuttonactive.xpm'
'icewm/themes/yast-installation/taskbar/windows.xpm' -> 'openSUSE/icewm/themes/yast-installation/taskbar/windows.xpm'
'icewm/themes/yast-installation/titleAB.xpm' -> 'openSUSE/icewm/themes/yast-installation/titleAB.xpm'
mkdir -p openSUSE/libreoffice/program
cp -r libreoffice/flat_logo.svg libreoffice/sofficerc libreoffice/shell openSUSE/libreoffice/program/
rsvg-convert libreoffice/intro.svg -o openSUSE/libreoffice/program/intro.png
optipng -o7 openSUSE/libreoffice/program/intro.png
** Processing: openSUSE/libreoffice/program/intro.png
500x200 pixels, 3x8 bits/pixel, RGB
Input IDAT size = 30342 bytes
Input file size = 30453 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 1		IDAT size = 29959
  zc = 9  zm = 8  zs = 0  f = 1		IDAT size = 29918
  zc = 9  zm = 9  zs = 0  f = 5		IDAT size = 29470
  zc = 9  zm = 8  zs = 0  f = 5		IDAT size = 29413
                               
Selecting parameters:
  zc = 9  zm = 8  zs = 0  f = 5		IDAT size = 29413

Output IDAT size = 29413 bytes (929 bytes decrease)
Output file size = 29488 bytes (965 bytes = 3.17% decrease)

mkdir -p openSUSE/wallpapers openSUSE/wallpapers/openSUSEdefault/contents/images
rsvg-convert raw-theme-drop/default-dark.svg -o openSUSE/wallpapers/openSUSEdefault/contents/images/default-dark.png
rsvg-convert raw-theme-drop/default.svg -o openSUSE/wallpapers/openSUSEdefault/contents/images/default.png
optipng -o5 openSUSE/wallpapers/openSUSEdefault/contents/images/default-dark.png
** Processing: openSUSE/wallpapers/openSUSEdefault/contents/images/default-dark.png
4096x4096 pixels, 3x8 bits/pixel, RGB
Input IDAT size = 181871 bytes
Input file size = 182210 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 2		IDAT size = 158472
  zc = 9  zm = 9  zs = 1  f = 2		IDAT size = 157990
                               
Selecting parameters:
  zc = 9  zm = 9  zs = 1  f = 2		IDAT size = 157990

Output IDAT size = 157990 bytes (23881 bytes decrease)
Output file size = 158065 bytes (24145 bytes = 13.25% decrease)

optipng -o5 openSUSE/wallpapers/openSUSEdefault/contents/images/default.png
** Processing: openSUSE/wallpapers/openSUSEdefault/contents/images/default.png
4096x4096 pixels, 3x8 bits/pixel, RGB
Input IDAT size = 139231 bytes
Input file size = 139498 bytes

Trying:
  zc = 9  zm = 9  zs = 0  f = 2		IDAT size = 113850
  zc = 9  zm = 8  zs = 0  f = 2		IDAT size = 113832
  zc = 9  zm = 9  zs = 1  f = 2		IDAT size = 113729
  zc = 9  zm = 8  zs = 1  f = 2		IDAT size = 113693
                               
Selecting parameters:
  zc = 9  zm = 8  zs = 1  f = 2		IDAT size = 113693

Output IDAT size = 113693 bytes (25538 bytes decrease)
Output file size = 113768 bytes (25730 bytes = 18.44% decrease)

sed "s:@VERSION@:16.0:g;s:@VERSION_NO_DOT@:`echo 16.0 | sed 's:\.::g'`:g" wallpapers/openSUSE.png.desktop.in > openSUSE/wallpapers/openSUSE`echo 16.0 | sed 's:\.::g'`.png.desktop
ln -sf openSUSE`echo 16.0 | sed 's:\.::g'`.png openSUSE/wallpapers/default.png
ln -sf openSUSE`echo 16.0 | sed 's:\.::g'`-dark.png openSUSE/wallpapers/default-dark.png
ln -sf openSUSEdefault/contents/images/default.png openSUSE/wallpapers/openSUSE`echo 16.0 | sed 's:\.::g'`.png
ln -sf openSUSEdefault/contents/images/default-dark.png openSUSE/wallpapers/openSUSE`echo 16.0 | sed 's:\.::g'`-dark.png
cp -p kde-workspace/metadata.json openSUSE/wallpapers/openSUSEdefault/metadata.json
cp -p raw-theme-drop/screenshot.png openSUSE/wallpapers/openSUSEdefault/screenshot.png
cp -p kde-workspace/metadata.json openSUSE/wallpapers/openSUSEdefault/metadata.json
mkdir -p openSUSE/yast_wizard
cp -a yast/* openSUSE/yast_wizard
mkdir -p openSUSE/plymouth
cp plymouth/config/plymouthd.defaults openSUSE/plymouth
📦[lkocman@osc branding_leap16]$ ls

```

**Comment by lkocman** [comment]
Shared with Marketing channel, no feedback in PR, seems like people are okay with it. Merging!

**Pull request merged**